### ISSUE TITLE
Incorrectly reporting orientation EXIF data

### ISSUE THREAD

**Issue by SaladFork** [tonytonyjan/exif#9]
I have a JPG image taken on a mobile device which added an `Orientation` EXIF tag since the image is rotated 90º counterclockwise.

Preview.app on Mac correctly identifies the orientation:
![](https://i.imgur.com/nA52reZ.png)

As does `exif`:

```bash
$ exif 'wrong-rotation.jpg' | grep Orientation
Orientation         |Right-top
```

And `EXIFR`:

```ruby
image_data = EXIFR::JPEG.new('wrong-rotation.jpg')
image_data.orientation  # => #<EXIFR::TIFF::Orientation:RightTop(6)>
```

However this library returns "unrotated":
```ruby
image_data = Exif::Data.new('wrong-rotation.jpg')
image_data.orientation  # => 0
```

**Comment by tonytonyjan** [comment]
@SaladFork Unfortunately, I am not able to reproduce this bug, please help me dumping your image by using `Exif::Data#dump`. If there is any private tag information that you don't want to put it here, you can drop me an email (a sample image as an attachment would be better).

**Comment by waffleau** [comment]
I'm seeing this issue too. I can reproduce with this image (found off Google): http://fredericiana.com/media/2013/monalisa.jpg

Orientation in EXIF comes up as 0. Can reproduce with a very simple use case:

```
exif = Exif::Data.new('monalisa.jpg').to_h
exif[:orientation]
```

**Comment by tonytonyjan** [comment]
@waffleau Thanks for reproducing, I did some test and it seems to be fixed in v2.0.0:

```ruby
# v1.0.1
Exif::Data.new('test/portrait.jpg') # => 0

# v2.0.0
Exif::Data.new(IO.read('test/portrait.jpg')) # => 6
```

Since I will not keep maintaining v1.x anymore, a pull request would be very appreciated. Otherwise, you can upgrade exif to v2.0.0 and read [CHANGELOG.md](https://github.com/tonytonyjan/exif/blob/ae0a2e1/CHANGELOG.md) for breaking changes.

Feel free to let me know if there is anything else I can help, and thanks for making exif better :)
